mappings/net/minecraft/structure/StructurePieceWithDimensions.mapping

```diff
@@ -15,3 +15,7 @@ CLASS net/minecraft/class_3418 net/minecraft/structure/StructurePieceWithDimensi
 	METHOD method_14839 (Lnet/minecraft/class_1936;Lnet/minecraft/class_3341;I)Z
 		ARG 1 world
 		ARG 2 boundingBox
+	METHOD method_37865 bury (Lnet/minecraft/class_1936;Lnet/minecraft/class_3341;I)Z
```

Comment:
This isn't exactly burying. This shifts the piece vertically to cover the lowest y in the x-z area specified by its bounding box.

You should best name this together with `method_14839`. These two methods are quite similar; they both choose the `hPos` for this piece. 14839 chooses by calculating the average height in the area for `hPos`, while this one finds the minimum height in the area for `hPos`. In the end, both shifts the bounding box vertically so the new `box.getMinY` is `hPos` (`+ deltaY` for this one)

Reply:
After this analysis, I believe this class is better named `ShiftableStructurePiece` as this is its main functionality; the two methods can be named `adjustToAverageHeight` and `adjustToMinHeight` (these are general ideas, feel free to come up with better ones)